pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8484-copy-configuration-operator.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="lkapanad" id="copy-configuration-operator">
+        <addColumn tableName="copy_answer_location">
+            <column name="user" type="varchar(30)" defaultValue="USER">
```

Comment:
Naming the column `user` could be misleading. How about we name this `user_type` instead?

Reply:
user_type suits better for sure.

Reply:
Rest all looks fine to me